yt/yt/server/queue_agent/queue_exporter.cpp

```diff
@@ -1240,7 +1283,17 @@ class TQueueExporter
         }
 
         // NB(apachee): Table lag is rounded up, since export period could've changed after the last successful export task.
-        auto exportPeriodSeconds = exportConfig->ExportPeriod.Seconds();
+        ui64 exportPeriodSeconds;
+        switch (exportSchedule.index()) {
+            case 0: {
+                exportPeriodSeconds = std::get<0>(exportSchedule).Seconds();
+                break;
+            }
+            case 1: {
+                exportPeriodSeconds = GetCronExpressionPeriod(std::get<1>(exportSchedule));
```

Comment:
Here as well, I would try to understand the value being exported here and make it correct. Even though this is a bit less important than the naming.